data/fwupd-offline-update.service.in

```diff
@@ -8,6 +8,16 @@ After=sysinit.target system-update-pre.target dbus.socket systemd-journald.socke
 Before=shutdown.target system-update.target
 
 [Service]
+# added automatically, for details please see
+# https://en.opensuse.org/openSUSE:Security_Features#Systemd_hardening_effort
+ProtectSystem=full
+ProtectHome=true
+ProtectHostname=true
+ProtectKernelTunables=true
+ProtectKernelLogs=true
+ProtectControlGroups=true
+RestrictRealtime=true
+# end of automatic additions
```

Comment:
I think get rid of the comment lines please; once it's upstream it's not really true any more :) Thanks.

Reply:
True :) - and done